BasicSteps/ProcessStep.cs

```diff
@@ -11,6 +13,12 @@
 
 namespace OpenTap.Plugins.BasicSteps
 {
+    public class EnvironmentVariable
```

Comment:
Should we make this a nested class inside ProcessStep? Or maybe we can just use `KeyValuePair<string,string>`?